doc/interop-test-descriptions.md

```diff
@@ -335,31 +346,32 @@ Procedure:
 Asserts:
 * call was successful
 * received SimpleResponse.username is in the json key file read from
-  FLAGS_service_account_key_file
-* received SimpleResponse.oauth_scope is in FLAGS_oauth_scope
+   ```--service_account_key_file```
```

Comment:
We should be using single tick \` instead of triple tick \`\`\`. I suggested wrong. See if swapping fixes it.